tutorials/basics/hello/hello2.py

```diff
@@ -14,10 +15,13 @@ class HelloMultiLangTest(rfm.RegressionTest):
     valid_systems = ['*']
     valid_prog_environs = ['*']
 
+    # rdmdocstart: set_sourcepath
     @run_before('compile')
     def set_sourcepath(self):
         self.sourcepath = f'hello.{self.lang}'
+    # rdmdocend: set_sourcepath
```

Comment:
```suggestion
    # rfmdocstart: set_sourcepath
    @run_before('compile')
    def set_sourcepath(self):
        self.sourcepath = f'hello.{self.lang}'
    # rfmdocend: set_sourcepath
```
If you change it `docs/tutorial_basics.rst` you should change it here as well.